core/src/test/resources/sql/sub-query.iq

```diff
@@ -3306,4 +3306,28 @@ EnumerableCalc(expr#0..7=[{inputs}], expr#8=[7782], expr#9=[CAST($t0):INTEGER NO
   EnumerableTableScan(table=[[scott, EMP]])
 !plan
 
+# [CALCITE-4846] IN-list that includes NULL converted to Values throws exception
+
+select * from "scott".emp where empno not in (null, 7782);
```

Comment:
just wondering if `(empno, depnto) in ((1,2), (3, null))` work or not. Do you think it is worth adding a test like that?

Reply:
Yes. I have added it.